admin_guide/pruning_resources.adoc

```diff
@@ -645,3 +645,13 @@ finished, the registry can be switched back to read-write mode by executing:
 $ oc env -n default dc/docker-registry REGISTRY_STORAGE_MAINTENANCE_READONLY-
 ----
 endif::[]
+
+[[pruning-cronjobs]]
+== Pruning Cron Jobs
+Cron Jobs are a technology preview feature, this means it lacks some capabilities to be
```

Comment:
Instead of `Cron Jobs are a technology preview feature, this means ... may not handle properly failed jobs.` Can you please put the following:
```
[IMPORTANT]
====
Cron Jobs is a Technology Preview feature only. It can perform pruning of successful jobs, but might not handle properly, the failed jobs.
ifdef::openshift-enterprise[]
Technology Preview features are not
supported with Red Hat production service level agreements (SLAs), might not be
functionally complete, and Red Hat does not recommend to use them for
production. These features provide early access to upcoming product features,
enabling customers to test functionality and provide feedback during the
development process.

For more information on Red Hat Technology Preview features support scope, see
https://access.redhat.com/support/offerings/techpreview/.
endif::[]
====
```